finalPrep/modZeroHero.js

```diff
@@ -56,16 +109,50 @@ var excuse = 'I think I forgot to lock up my 1992 Toyota Coralla. Be right back.
 //   - luckyNumbers (array)
 //   - address (object with following key/values: number , street , state, zip)
 
+var scaryMonster = {
+  name: "NoroMan",
+  smell: "Thru-hiker",
+  weight: "200",
+  citiesDestroyed: ["Neel's Gap", "Hot Springs", "Waynesboro"],
+  luckyNumbers: [1921, 1948, 2017, 2018],
+  address: {
+    number: 4125,
+    street: "Coxey Brown Road",
+    state: "Maryland",
+    zip: 21773
+  }
+
+}
 
 // Create a new class called SuperHero
 // - Your class should have the following DYNAMIC values
-//   - name 
+//   - name
 //   - superpower
-//   - age 
+//   - age
 // - Your class should have the following STATIC values
 //   - archNemesis, assigned to "The Syntax Error"
 //   - powerLevel = 100
-//   - energyLevel = 50 
+//   - energyLevel = 50
+
+
+class SuperHero {
+  constructor(name, superpower, age) {
+    this.name = name;
+    this.superpower = superpower;
+    this.age = age;
+    this.archNemesis = "The Syntax Error";
+    this.powerLevel = 100;
+    this.energyLevel = 50;
+  }
+  sayName:function() {
+    console.log("name");
+  }
+  maximizeEnergy:function() {
+    console.log(energyLevel === 1000);
+  }
+  gainPower(); {}
+    console.log('n' + powerLevel);
+}
```

Comment:
🚨 This needed to "take an argument"
🚨 What is the behavior for truly increasing the "power level"? Is this function increasing power level? What is this function doing?